src/App.tsx

```diff
@@ -7,18 +7,111 @@ import './App.css'
 import FocusManager from './components/FocusManager'
 
 import AppRoot from './AppRoot'
+import {
+  ScreenReader,
+  AriaScreenReader,
+  SpeechSynthesisTextToSpeech,
+  NullTextToSpeech,
+  TextToSpeech,
+} from './utils/ScreenReader'
 
 /* istanbul ignore next - unsure how to test */
 window.oncontextmenu = (e: MouseEvent): void => {
   e.preventDefault()
 }
 
-const App = () => (
-  <BrowserRouter>
-    <FocusManager>
-      <Route path="/" component={AppRoot} />
-    </FocusManager>
-  </BrowserRouter>
-)
+export interface Props {
+  tts?: {
+    enabled: TextToSpeech
+    disabled: TextToSpeech
+  }
+}
+
+const App = ({
+  tts = {
+    enabled: new SpeechSynthesisTextToSpeech(),
+    disabled: new NullTextToSpeech(),
+  },
+}: Props) => {
+  const [screenReaderEnabled, setScreenReaderEnabled] = useState(false)
+  const [screenReader, setScreenReader] = useState<ScreenReader>(
+    new AriaScreenReader(tts.disabled)
+  )
+
+  /* istanbul ignore next - need to figure out how to test this */
```

Comment:
I'm having flashbacks from trying to test the previous keyboard shortcut (⌘-K) which we never figured out how to do before we eventually removed the feature. cc @ben

Reply:
One way to do it is to move these out of `App` and into something a bit more testable, then maybe have a much smaller untested bit here.